api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirTaskDaoImpl.java

```diff
@@ -23,27 +23,19 @@
 import lombok.AccessLevel;
 import lombok.Setter;
 import org.hibernate.Criteria;
-import org.hibernate.SessionFactory;
-import org.hibernate.criterion.Criterion;
```

Comment:
You should only remove imports that aren't actually used.

Reply:
Hey @ibacher more advice on this fhirTaskDaoImpl  class coz it seems to  be abstracted from other implemented classes

Reply:
What can I advise you on?

Reply:
I actually did what you told me before but same problem arising, could i be missing some steps, let me look through it again